sdks/python/apache_beam/coders/coders_test_common.py

```diff
@@ -63,6 +65,34 @@ class CodersTest(unittest.TestCase):
   # These class methods ensure that we test each defined coder in both
   # nested and unnested context.
 
+  # Common test values representing Python's built-in types.
+  values_deterministic: List[Any] = [
```

Comment:
Nit: `values_deterministic` is kind of vague, `values` even more so. Can we rename to `test_values*`?

Reply:
done